enterprise/frontend/src/metabase-enterprise/resource_downloads/resource_downloads_plugin.ts

```diff
@@ -1,7 +1,25 @@
+import { match } from "ts-pattern";
+
 import { PLUGIN_RESOURCE_DOWNLOADS } from "metabase/plugins";
 
-// TODO: implement real logic
 /**
  * Returns if 'download results' on cards and pdf exports are enabled in public and embedded contexts.
  */
-PLUGIN_RESOURCE_DOWNLOADS.areDownloadsEnabled = () => true;
+PLUGIN_RESOURCE_DOWNLOADS.areDownloadsEnabled = ({
```

Comment:
There's actually a bug here, I'm not checking for the whitelabel feature, working on it at the moment